date_guesser/constants.py

```diff
@@ -1,4 +1,3 @@
-from collections import namedtuple
 from enum import Enum, IntEnum
```

Comment:
if you're changing this file anyways, can you remove the import of `Enum`? it is unused. i'll set up an automated lint after merging this.